src/openfermion/utils/_trotter_exp_to_qgates.py

```diff
@@ -167,6 +168,10 @@ def pauli_exp_to_qasm(qubit_operator_list,
             QubitOperators to be exponentiated
         evolution_time (float): evolution time of the operators in
             the list
+        qubit_list: a list of names for qubits (otherwise they will be labeled
```

Comment:
Can we include the type a bit more explicitly?  Is it a list of strings with length of the number of qubits for example?

Reply:
Perhaps even a little elaboration as to its use and sanity checking on its size / conditions for proper operation.

Reply:
Completely fair point. I'm not entirely happy with the way this is implemented to be honest. The issue is that if I define a qubit in a piece of code I often want it to have a label that is a valid variable name, but when I define a qubit or fermion operator in a system I almost always just label with a number. This was my way of trying to bridge between the two, but it feels a bit hacky.

As it is, I'll insert a sanity check on the size with a detailed error message and add a more detailed description. But there may be a better way to deal with this.